nengo_loihi/tests/test_loihi_api.py

```diff
@@ -1,7 +1,10 @@
 import numpy as np
 import pytest
 
+from nengo.utils.numpy import rms
```

Comment:
From the perspective of Nengo Loihi, Nengo is a third-party dependency like all the rest, so it should go in the upper block with numpy and pytest.